zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DbReader.scala

```diff
@@ -73,14 +73,20 @@ object DbReader {
     override def getk8sNodepoolsToDeleteCandidate: Stream[F, NodepoolToScan] =
       activeNodepoolsQuery.stream.transact(xa)
 
-    override def markNodepoolAndAppStatusDeleted(id: Long): F[Unit] = {
+    override def markNodepoolAndAppStatusDeleted(nodepoolId: Long): F[Unit] = {
       val res = for {
-        _ <- updateNodepoolStatus(id, "DELETED").run
-        _ <- updateAppStatusForNodepoolId(id).run
+        _ <- updateNodepoolStatus(nodepoolId, "DELETED").run
+        _ <- updateAppStatusForNodepoolId(nodepoolId, "DELETED").run
       } yield ()
       res.transact(xa)
     }
 
-    def markNodepoolError(id: Long): F[Unit] = updateNodepoolStatus(id, "ERROR").run.transact(xa).void
+    def markNodepoolAndAppError(nodepoolId: Long): F[Unit] = {
+      val res = for {
+        _ <- updateNodepoolStatus(nodepoolId, "ERROR").run
+        _ <- updateNodepoolStatus(nodepoolId, "ERROR").run
```

Comment:
^^ looks like copy pasta 😛

Reply:
oh man, I was thinking the function is so simple and doesn't need a separate unit test for itself...apparently I was wrong